content/v2.md

```diff
@@ -193,6 +193,37 @@ were created in and notices based on these objects will be delivered with
 the proper locale. i.e. if an issue is created with the "es" locale, all
 email to the creator will be delivered with Spanish translations.
 
+## Terms of Use
```

Comment:
I think the info you added should be incorporated into the Data License section that already exists below, rather than a new section before that.  And at the top of the document something like:
```
Use of the API documented here is subject to the terms of our Data License.
```
And have a link to the Data License section.